elyra/pipeline/component.py

```diff
@@ -271,7 +271,7 @@ def read_component_definition(self, component_id: str, location: str) -> str:
         res = requests.get(location)
         if res.status_code != HTTPStatus.OK:
             self.log.error (f'Invalid location for component: {component_id} -> {location} (HTTP code {res.status_code})')  # noqa: E211 E501
```

Comment:
We should remove the `# noqa:` directive and split the line.
Also, please use double quotes for the f-string and (similar to above) convert to a warning log.